pkg/api/validation/validation.go

```diff
@@ -2272,6 +2276,12 @@ func ValidatePodSpec(spec *api.PodSpec, fldPath *field.Path) field.ErrorList {
 		allErrs = append(allErrs, ValidateHostAliases(spec.HostAliases, fldPath.Child("hostAliases"))...)
 	}
 
+	if len(spec.PriorityClassName) > 0 {
```

Comment:
The rest handler cannot depend on the admission plug-in being present, and by the time the object reaches validation and rest handling, there is no way to know whether the field was set by a user or an admission plug-in